src/net.h

```diff
@@ -312,6 +314,7 @@ class CConnman
      * call the function without a parameter to avoid using the cache.
      */
     std::vector<CAddress> GetAddresses(CNode& requestor, size_t max_addresses, size_t max_pct);
+    std::vector<CAddress> GetCurrentBlockRelayOnlyConns() const;
```

Comment:
This can be private. Can you also add a short doxygen comment eg "Return vector of current BLOCK_RELAY peers"

Reply:
Thanks. [Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-706081630).